x-pack/plugins/security_solution/public/flyout/document_details/right/components/header_title.tsx

```diff
@@ -82,6 +97,15 @@ export const HeaderTitle: FC = memo(() => {
         <EuiFlexItem grow={false}>
           <RiskScore />
         </EuiFlexItem>
+        {scopeId !== TableId.rulePreview && (
```

Comment:
Just want to bring your attention to this PR that uses new expandable flyout in create rule workflow. Once it is merged you will have access to `isPreview` here